app/javascript/components/Header/index.jsx

```diff
@@ -107,14 +106,17 @@ const MenuItems = () =>
         <MenuItem href="/pages/informationevent">Attend an event</MenuItem>
     </>
 
-const SiteHeader = () => {
+const SiteHeader = ({logo}) => {
     const [menuOpen, toggleMenu] = useState(false)
+    console.log({logo})
 
     return(
         <Outer>
             <Inner>
                 <MastheadLink href="/">
-                    <Logo src={logo} alt="Buckinghamshire County Council"/>
+                    
+                        {logo && <Logo src={logo} alt="Buckinghamshire County Council"/>}
```

Comment:
We only display the logo if it's a valid logo. If they prefer we can have a default image instead nothing.

Reply:
Nice! We may wish to make the `alt` text dynamic as well, given that the logo now might not be Bucks' logo - what do you think?

Reply:
Yes! I wasn't sure, I thought we would keep only for bucks. I will do the updates :D